packages/design-system-scripts/gulp/docs/generatePages/generatePages.js

```diff
@@ -201,28 +206,48 @@ module.exports = async function generatePages(sourceDir, docsDir, options, chang
     )
   );
 
-  // Merge both sets of page sections into a single array.
+
+  const addCmsdsLink = function (page) {
```

Comment:
I dont think this syntax would work, as its combining a named function declaration with a variable declaration. You can take a look at the other functions in the same file as examples (i.e. `function changedFilter()`). These are two examples of how a function can be defined:

```
const addCmsdsLink = () => {};
function addCmsdsLink() {}
```

Reply:
I would also move this function code outside of the `generateDocPages` function. It can still be defined in the file, but it's cleaner if you put it elsewhere.